libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/parametersTab/identityselector.tsx

```diff
@@ -54,14 +54,22 @@ export const IdentitySelector = (props: IdentitySelectorProps) => {
 
   const [selectedKey, setSelectedKey] = useState(selectedIdentity);
 
+  const isIdentityInLogicApp = useMemo(() => {
+    return isIdentityPresentInLogicApp(selectedKey, identity);
+  }, [identity, selectedKey]);
+
+  const isIdentityWithLogicApp = useMemo(() => {
+    return isIdentityAssociatedWithLogicApp(identity);
+  }, [identity]);
+
   useEffect(() => {
```

Comment:
It looks like `identity` not being memoized was the issue here (causing it to run on every render)
If that's true shouldn't we be memoizing `identity` on line 45?
Otherwise this still runs into the same issue

Reply:
Thank you so much, made the fix and it worked